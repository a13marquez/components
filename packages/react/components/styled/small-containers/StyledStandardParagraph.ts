import styled from 'styled-components';
import { StandardStyles } from '../shared-styles';

export const StyledStandardParagraph = styled.p`
${StandardStyles}
`