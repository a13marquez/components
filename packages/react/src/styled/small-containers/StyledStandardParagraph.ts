import styled from 'styled-components';
import shared from '../shared-styles';

const StyledStandardParagraph = styled.p`
  ${shared.StandardStyles}
`;

export default StyledStandardParagraph