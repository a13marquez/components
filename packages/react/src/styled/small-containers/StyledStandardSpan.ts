import styled from 'styled-components';
import shared from '../shared-styles';

const StyledStandardSpan = styled.span`
  ${shared.StandardStyles}
`;

export default StyledStandardSpan;