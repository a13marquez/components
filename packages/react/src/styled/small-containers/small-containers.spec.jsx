import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';

import StyledStandardParagraph from './StyledStandardParagraph';
import StyledStandardSpan from './StyledStandardSpan';

test('It renders Styled Standard Paragraph', () => {
  const StandardParagraph = mount(<StyledStandardParagraph />);
  expect(StandardParagraph).toMatchSnapshot();
});

test('It renders Styled Standard Span', () => {
  const StandardSpan = mount(<StyledStandardSpan />);
  expect(StandardSpan).toMatchSnapshot();
});


test('Styled Standard Paragraph have the correct styles', () => {
  const StandardParagraph = mount(<StyledStandardParagraph />);
  expect(StandardParagraph).toHaveStyleRule('margin', '0');
  expect(StandardParagraph).toHaveStyleRule('padding', '0');
  expect(StandardParagraph).toHaveStyleRule('border', '0');
  expect(StandardParagraph).toHaveStyleRule('vertical-align', 'baseline');
  expect(StandardParagraph).toHaveStyleRule('font', '100%');
});

test('Styled Standard Span have the correct styles', () => {
  const StandardSpan = mount(<StyledStandardSpan />);
  expect(StandardSpan).toHaveStyleRule('margin', '0');
  expect(StandardSpan).toHaveStyleRule('padding', '0');
  expect(StandardSpan).toHaveStyleRule('border', '0');
  expect(StandardSpan).toHaveStyleRule('vertical-align', 'baseline');
  expect(StandardSpan).toHaveStyleRule('font', '100%');
});