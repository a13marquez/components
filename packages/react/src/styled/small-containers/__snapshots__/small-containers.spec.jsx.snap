// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders Styled Standard Paragraph 1`] = `ReactWrapper {}`;

exports[`It renders Styled Standard Span 1`] = `ReactWrapper {}`;
